File: src/minigames/guess_it/src/main/java/work/lclpnet/ap2/game/guess_it/data/ChallengeMessenger.java
package work.lclpnet.ap2.game.guess_it.data;

import net.minecraft.network.chat.Component;
import work.lclpnet.kibu.translate.text.TranslatedText;

public interface ChallengeMessenger {

    void task(TranslatedText task);

    void options(Component[] options);
}

File: src/minigames/guess_it/src/main/kotlin/work/lclpnet/ap2/game/guess_it/data/ChallengeMessenger.kt
package work.lclpnet.ap2.game.guess_it.data

import net.minecraft.network.chat.Component
import work.lclpnet.kibu.translate.text.TranslatedText

interface ChallengeMessenger {

    fun task(task: TranslatedText)

    fun options(vararg options: Component)
}
